src/Microsoft.AspNetCore.Server.IntegrationTesting/breakingchanges.netcore.json

```diff
@@ -0,0 +1,78 @@
+[
+    {
+      "TypeId": "public abstract class Microsoft.AspNetCore.Server.Testing.ApplicationDeployer : Microsoft.AspNetCore.Server.Testing.IApplicationDeployer",
```

Comment:
These removals are due to rename of the namespace from M.A.Server.Testing to M.A.Server.IntegrationTesting.